youcan-shop/cod-theme-3
# YSHOP2-939 > theme 3 > Empty cart page & empty search results

## JIRA Ticket

[YSHOP2-939](https://youcanshop.atlassian.net/browse/YSHOP2-939)

## QA Steps

-  [ ]  check the new updates on search page & cart page & collection page

## Note

I also add the empty result page UI to collection page

[YSHOP2-939]: https://youcanshop.atlassian.net/browse/YSHOP2-939?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
